tests/v2_tests/test_add_documents.py

```diff
@@ -395,384 +349,340 @@ def test_batching_add_docs(self):
-                mock__post = mock.MagicMock()
-                mock__post.return_value = dict()
-
-                @mock.patch("marqo._httprequests.HttpRequests.post", mock__post)
-                def run():
-                    res = self.client.index(self.generic_test_index_name).add_documents(
-                        auto_refresh=auto_refresh, documents=docs, client_batch_size=client_batch_size,
-                        tensor_fields=["Title", "Description"])
-                    if client_batch_size is not None:
-                        assert isinstance(res, list)
-                        assert len(res) == math.ceil(docs_to_add / client_batch_size)
-                        # should only refresh on the last call, if auto_refresh=True
-                        assert all([f'refresh=false' in d[1]['path'] for d in
-                                    mock__post.call_args_list][:-1])
-                        if auto_refresh:
-                            assert [f"{self.generic_test_index_name}/refresh" in d[1]['path']
-                                    for d in mock__post.call_args_list][-1]
-                    else:
-                        assert isinstance(res, dict)
-                        # One huge request is made, if there is no client_side_batching:
-                        assert all([len(d[1]['body']["documents"]) == docs_to_add for d in mock__post.call_args_list])
-
-                    assert all(['batch' not in d[1]['path'] for d in mock__post.call_args_list])
-
-                    assert all(['processes' not in d[1]['path'] for d in mock__post.call_args_list])
-
-                    return True
+        for client_batch_size in batches:
+            mock__post = mock.MagicMock()
+            mock__post.return_value = dict()
+
+            @mock.patch("marqo._httprequests.HttpRequests.post", mock__post)
+            def run():
+                res = self.client.index(self.generic_test_index_name).add_documents(
+                    documents=docs, client_batch_size=client_batch_size,
+                    tensor_fields=["Title", "Description"]
+                )
+                if client_batch_size is not None:
+                    assert isinstance(res, list)
+                    assert len(res) == math.ceil(docs_to_add / client_batch_size)
+                    # should only refresh on the last call, if auto_refresh=True
+                else:
+                    assert isinstance(res, dict)
+                    # One huge request is made, if there is no client_side_batching:
+                    assert all([len(d[1]['body']["documents"]) == docs_to_add for d in mock__post.call_args_list])
+
+                assert all(['batch' not in d[1]['path'] for d in mock__post.call_args_list])
+
+                assert all(['processes' not in d[1]['path'] for d in mock__post.call_args_list])
+
+                return True
 
                 assert run()
 
-    def test_add_lists_non_tensor(self):
```

Comment:
can we comment these out rather than delete it? we want to eventually fix and make these tests work